left_pan_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: left_pan_joint
    joint_speed: 0.8
    motor:
        id: 0
        init: 512
        min: 0
        max: 1023

left_tilt_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: left_tilt_joint
    joint_speed: 0.8
    motor:
        id: 1
        init: 512
        min: 0
        max: 1023

mid_pan_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: mid_pan_joint
    joint_speed: 0.8
    motor:
        id: 2
        init: 512
        min: 0
        max: 1023

mid_tilt_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: mid_tilt_joint
    joint_speed: 0.8
    motor:
        id: 3
        init: 512
        min: 0
        max: 1023

right_pan_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: right_pan_joint
    joint_speed: 0.8
    motor:
        id: 4
        init: 512
        min: 0
        max: 1023

right_tilt_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: right_tilt_joint
    joint_speed: 0.8
    motor:
        id: 5
        init: 512
        min: 0
        max: 1023
